src/graphql/Bug.graphql

```diff
@@ -269,6 +269,12 @@ input BugSearch {
   # Search the "Status Whiteboard" field on bugs for a substring. Works the same as the `summaries`
   # field, but searches the Status Whiteboard field.
   whiteboards: [String]
+
+  # The field in the bug to be used as advanced query parameters
+  f1: String,
+
+  # The conditional of the field stated in f1 written in small letters to be used as advanced query
+  o1: String,
```

Comment:
Is this supposed to be an enum?

```graphql
enum Condition {
  IS_EMPTY
  IS_NOT_EMPTY
  CONTAINS_ALL_OF
  CONTAINS_ANY_OF
  IS_LESS_THAN
  IS_GREATER_THAN
}

o1: Condition,
```

Reply:
I initially think so too. But when i google, i found out that there are 27 possibilities of the value as shown below in the dropdown. Would it be ok to have enum?
<img width="779" alt="screen shot 2018-07-26 at 7 55 36 pm" src="https://user-images.githubusercontent.com/24544912/43261069-ff244b58-910d-11e8-8074-787199dc8cca.png">

Reply:
Sure, I think it's still ok to go with an enum.